src/main/java/com/iridium/iridiumteams/listeners/SpawnerSpawnListener.java

```diff
@@ -27,7 +31,25 @@ public void onCreatureSpawn(SpawnerSpawnEvent event) {
             if (!teamEnhancement.isActive(spawnerEnhancement.type)) return;
             if (data == null) return;
 
-            spawner.setSpawnCount(data.spawnCount);
+            HashSet<SpawnerSupport> spawnerSupportList = iridiumTeams.getSupportManager().getSpawnerSupport();
+
+            if (spawnerSupportList.isEmpty()) {
+                if (iridiumTeams.getConfiguration().multiplicativeSpawners)
+                    spawner.setSpawnCount(spawner.getSpawnCount() * data.spawnCount);
+                else spawner.setSpawnCount(data.spawnCount);
+            }
+
+            else {
+                for(SpawnerSupport spawnerSupport : spawnerSupportList) {
+                    if (!spawnerSupport.isStackedSpawner(spawner.getBlock())) continue;
+                    if (iridiumTeams.getConfiguration().multiplicativeSpawners) {
+                        spawner.setSpawnCount(spawnerSupport.spawnerSpawnAmount(spawner) * data.spawnCount);
+                        continue;
+                    }
+                    spawner.setSpawnCount(spawnerSupport.spawnerStackAmount(spawner) * data.spawnCount);
+                }
+            }
+
```

Comment:
Not sure about this either, we want a method to get the spawners in the stack and then smthn like

```java
int stackSize = ...
int modifier = iridiumTeams.getConfiguration().multiplicativeSpawners ? stackSize : 1;
spawner.setSpawnCount(modifier * data.spawnCount);
```

We also probabbly want some configs for the max spawner size count, so if somone stacks 100x spawners they dont spawn 600 zombies

Reply:
if someone is stacking spawners, they're using a stacker plugin, and those plugins typically have limits on the spawns themselves or stack the entities (so outside of scope as far as im concerned)
also important to mention - this doesn't apply if they're not using a stacker plugin, as it will just use the config value, in which case we shouldn't be limiting that anyway (if a server owner wants to be stupid, let them)

Reply:
Yes, but we are overriding it here no? because of the island boosters for spawners no?

Reply:
uh
tbh clarification on how the spawner enhancement should function would help a lot here

Reply:
uh
tbh clarification on how the spawner enhancement should function would help a lot here